ReactiveCocoa/Swift/Signal.swift

```diff
@@ -331,6 +331,21 @@ extension SignalType {
 	}
 }
 
+extension SignalType {
+	/// Merges the given signals into a single `Signal` that will emit all values
+	/// from each of them, and complete when all of them have completed.
+	public static func mergeSignals(signals: [Signal<Value, Error>]) -> Signal<Value, Error> {
```

Comment:
I changed the name.
Unfortunately I couldn't get the `SequenceType` change to compile:

```swift
public static func merge<T: SignalType, S: SequenceType where S.Generator.Element == T, T.Value == Self.Value, T.Error == Self.Error>
```

That's fine, but then it can't find a definition of `flatten`... and I'm having a hard time figuring out why because `SourceKit` keeps crashing :fire: